geth/jail/extensions/console/console.go

```diff
@@ -123,8 +125,8 @@ func convertArgs(argumentList []otto.Value) []interface{} {
 
 // handleConsole takes the giving otto.Values and transform as
 // needed into the appropriate writer.
-func handleConsole(tag string, writer io.Writer, args []otto.Value) {
-	fmt.Fprintf(writer, tag, formatForConsole(args))
+func handleConsole(format string, writer io.Writer, args []otto.Value) {
```

Comment:
What do you think about `format` switching places with `writer` anyway? This would be more compliant with `Fprintf` and thus easier to read in my opinion.